microprofile/4.1/annotations/1.3/TCKResults.adoc

```diff
@@ -0,0 +1,83 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of Jakarta Annotations 1.3.
+
+== Open Liberty 21.0.0.9 - Jakarta Annotations 1.3 Certification Summary
```

Comment:
Why do you need to list annotation tck separately instead of merging it under the other file?

Reply:
I don't have to, if you don't want it separate.  I'd never run it before, so there wasn't already a place for it in my EE8 spec source docs, and I noted that all of the other MP ones are separate docs.  I considered breaking the CDI one into a separate doc, too, so that it was separated by how they were tested.

Any arrangement is fine with me, though.